modules/subscription/provider.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  required_providers {
+    azuredevops = {
+      source  = "microsoft/azuredevops"
+      version = "0.1.8"
```

Comment:
```suggestion
```

Reply:
There's still a bug in the provider that means any version above 0.20 doesn't work for us https://github.com/microsoft/terraform-provider-azuredevops/issues/199

Reply:
ok